proto/vtctlservice.proto

```diff
@@ -156,6 +156,12 @@ service Vtctld {
   // PlannedReparentShard or EmergencyReparentShard should be used in those
   // cases instead.
   rpc InitShardPrimary(vtctldata.InitShardPrimaryRequest) returns (vtctldata.InitShardPrimaryResponse) {};
+  // MoveTablesCreate creates a workflow which moves one or more tables from a
+  // source keyspace to a target keyspace.
+  rpc MoveTablesCreate(vtctldata.MoveTablesCreateRequest) returns (vtctldata.WorkflowStatusResponse) {};
```

Comment:
There is, but we return the status/progress command response (same basic thing that vtctlclient command does). ```
❯ ./202_move_tables.sh
{
  "table_copy_state": {},
  "shard_streams": {
    "customer/0": {
      "streams": [
        {
          "id": 26,
          "tablet": "zone1-0000000200",
          "source_shard": "commerce/0",
          "position": "MySQL56/9d10e6ec-07a0-11ee-ae73-8e53f4cf3083:1-97",
          "status": "Copying",
          "info": "VStream Lag: 0s"
        }
      ]
    }
  }
}
Waiting for the commerce2customer workflow in the customer keyspace to finish the copy phase...
The commerce2customer workflow in the customer keyspace is now running. Copy phase completed at gtid 9d10e6ec-07a0-11ee-ae73-8e53f4cf3083:1-97.
```

Reply:
imo, there's nothing wrong with defining a (nearly-) duplicate message type; if we ever add a field that only applies to one it's much easier to do that rather than split them out (which would essentially require defining an entirely different — with an entirely different name — RPC) after the fact

obligatory steel man: counterpoint that maintaining two identical data structures puts us at risk of drift, but (countering my counterpoint) that is fixed via a bug-fix rather than a breaking change requiring a deprecation cycle, so i could go either way with a _slight_ preference toward duplication

Reply:
I could embed `vtctldata.WorkflowStatusResponse` so that we can add additional info if we want for MoveTables creation.